frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/MfaHandler.java

```diff
@@ -151,7 +152,7 @@ public APIGatewayProxyResponseEvent handleRequestWithUserContext(
                                 .getClient()
                                 .map(ClientRegistry::getClientID)
                                 .orElse(AuditService.UNKNOWN),
-                        AuditService.UNKNOWN,
+                        String.valueOf(pair("journey-type", journeyType)),
```

Comment:
What's the behaviour if journeyType is null?  Is there a test to show that this doesn't generate a null pointer exception?

Reply:
No, it will just pass the null value, it will not throw out an exception :)